### PR TITLE
Bug fix html creator + html documenter

### PR DESCRIPTION
**Pull request by liuting1014**
1. Copied source code from html creator and fixed html entity escaping bug
2. Fix bug in html documenter to render function param description

### CODE REVIEW

File: src/html/html-creator/element.js [hunk 1]
```diff
@@ -18,22 +20,9 @@ const parseContent = (content, createElementMethod) => {
     if (content && content.constructor === Array) {
         return content.map(element => createElementMethod(element)).join('');
     }
-    return escapeHtmlEntities(content) || '';
+    return htmlEntities.escape(content) || '';
```
Comment (johanblumenberg-symphony) [inline src/html/html-creator/element.js:23]: How does it handle undefined?

```suggestion
    return content ? htmlEntities.escape(content) : '';
```

File: src/html/html-creator/element.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * @param {String} type - The HTML Tag type
+ * @param {Object} applyAttributes - The HTML attributes to be added to the tag
+ * @param {String/Array} content - The content of the tag. Can be either a string or an array of elements
+ */
+const create = ({ type, attributes, content }) => {
+    if (type) {
+        return (content) ?
+            `<${type}${applyAttributes(attributes)}>${parseContent(content, create)}</${type}>` : `<${type}${applyAttributes(attributes)} />`;
+    }
+    return content;
```
Comment (johanblumenberg-symphony) [inline src/html/html-creator/element.js]: Does this one need to be escaped as well?

File: src/html/html-creator/element.js (hunk 1 above)
Comment (liuting1014) [reply src/html/html-creator/element.js:23]: It's already handled in `create()`.  So basically `parseContent()` will never receive undefined as content.

File: src/html/html-creator/element.js (hunk 2 above)
Comment (liuting1014) [reply src/html/html-creator/element.js:37]: I don't think so. The content will always be passed to `parseContent()` and it will be escaped there.

**Review by johanblumenberg-symphony — APPROVED** [review]

**Pull request merged**